### PR TITLE
Add bump config to track debian:buster-slim version

### PR DESCRIPTION
**Pull request by wader**

### CODE REVIEW

**Comment by wader** [comment]
Just have to remember to add a `BUMP_TOKEN` secret before merge

**Comment by wader** [comment]
I usually prefer to use a specific version tags as possible to keep things as reproducible as possible that is the reason i built bump. One bad thing about doing this is that if someone forks or copies the Dockerfile they will be on the same version until manual change.

File: .github/workflows/bump.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  workflow_dispatch:
+
+jobs:
+  version_update:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - uses: wader/bump/action@master
+        env:
+          GITHUB_TOKEN: ${{ secrets.BUMP_TOKEN }}
```
Comment (hemberger) [inline .github/workflows/bump.yml:16]: Can this not use the automatically supplied `secrets.GITHUB_TOKEN`?
https://docs.github.com/en/actions/reference/authentication-in-a-workflow#about-the-github_token-secret

Comment (wader) [reply .github/workflows/bump.yml:16]: Yes sorry, was a note to myself to note forgot to do it.

Comment (wader) [reply .github/workflows/bump.yml:16]: https://docs.github.com/en/actions/reference/events-that-trigger-workflows#triggering-new-workflows-using-a-personal-access-token

Comment (hemberger) [reply .github/workflows/bump.yml:16]: Thanks for the reminder. That is certainly less than ideal in this case. Oh well!

**Pull request merged**